libV2/cache/backend/memory.js

```diff
@@ -0,0 +1,45 @@
+class MemoryCache {
+    constructor() {
+        this._data = {};
+        this._shards = {};
+    }
+
+    async getKey(key) {
```

Comment:
Does it have to be `async`? I see multiple `async` functions here without `await`, any specific reason, not an issue though!

Reply:
`async` here is mostly for the convenience. Since both memory and redis backends need the same API so they can be swapped out, memory needs needs to return a promise as well. Using `async` allows me to wrap the return value in a resolved promise automatically

Reply:
Fair enough 👍